packages/core/stories/spinner/spinner.doc.stories.mdx

```diff
@@ -34,10 +34,48 @@ Spinners have two sizes, `default` and `large`.
 
 Set the `size` prop to `"large"` to use the large Spinner.
 
+Large Spinners can be beneficial for full screen loading experiences; while default Spinners can be reserved for widget loading experiences.
+
 <Canvas>
   <Story id="core-spinner--large" />
 </Canvas>
 
+## Loading
+
+Center Spinner horizontally and vertically over a page or content that is loading if it takes between 1-9 seconds.
+
+<Canvas>
+  <Story id="core-spinner--loading" />
+</Canvas>
+
+## Partial Loading
```

Comment:
Is Partial loading going to be the official term?

Reply:
I feel like I find both not to be ideal. I think we can keep partial loading for now